chairing.py

```diff
@@ -19,43 +17,64 @@
 
 black = CMYKColor(0, 0, 0, 1)
 
+SLIDO_EVENT_CODE = "rsecon23"
+"""The code to access the main Q&A portal for the conference from slido.com"""
+SLIDO_EVENT_HASH = "mwHnJYH3zt11GVLKnUTfvu"
+"""The specific hash sequence that will get you directly to the event page or wall"""
+SLIDO_EVENT_LINK = f"https://app.sli.do/event/{SLIDO_EVENT_HASH}"
+"""The Q&A portal for the whole event. Specific room selection will be required."""
+SLIDO_EVENT_WALL = f"https://wall.sli.do/event/{SLIDO_EVENT_HASH}"
+"""The wall presentation page for the whole event. Specific room selection will be required."""
+SLIDO_ROOM_WALL_MAP = {
+    "GH043": "https://wall.sli.do/event/mwHnJYH3zt11GVLKnUTfvu?section=7c7a20b8-7014-4030-8e3a-08ae40150fa9",
+    "GH049": "https://wall.sli.do/event/mwHnJYH3zt11GVLKnUTfvu?section=b7b4e005-652e-408e-a7dd-4458980bcfb0",
+    "GH037": "https://wall.sli.do/event/mwHnJYH3zt11GVLKnUTfvu?section=a97ae336-63cb-461c-977c-ec5a03a06b12",
+    "GH001": "https://wall.sli.do/event/mwHnJYH3zt11GVLKnUTfvu?section=7f16f6af-3588-4c53-980d-93880e132579",
+}
+"""The wall presentation page for each specific room."""
+
 
 BASE_DOC = [
     "Dear {chair},",
     "Many thanks for agreeing to chair the <b>{session}</b> session at RSECon23. Please find below some guidance on your role as session chair.",
     ("Before the session", styles["Heading1"]),
-    "Your session starts at <b>{start_time} on {day}</b>. Please arrive at room <b>{room}</b> 10 minutes in advance of your session. Your room will have a volunteer present to help the event run smoothly—they will manage such things as moderating the Sli.do questions, ensuring speakers have microphones, and loading up presentations onto the PC to display. Please introduce yourself to your volunteer before the start of the session.",
-    "The speakers in your session should also be present before the start of the session to introduce themselves. <b>Please confirm the correct pronunciation of their names</b>, especially if the name is unfamiliar to you.",
-    "In case you need it, the username for the PC is <b>{login_username}</b>, and the password is <b>{login_password}</b>.",
+    "Your session starts at <b>{start_time} on {day}</b>. Please arrive at room <b>{room}</b> 10 minutes in advance of your session. Your room will have a volunteer and stream director present to help the event run smoothly—they will manage such things as moderating the Slido questions, ensuring speakers have microphones, loading up presentations onto the PC to display, and ensuring remote presenters are connected to the Zoom call and audible. Please introduce yourself to your volunteer and stream director before the start of the session.",
```

Comment:
There will only be a stream director if the session has a remote presenter—and the stream director will not be in the room; they'll be sitting somewhere in SlidesLive's offices.

There will be two volunteers, one who is primarily responsible for monitoring the chat with SlidesLive and ensuring that the audio and video are transmitting correctly.